technical-report/models/datatypes.wsd

```diff
@@ -0,0 +1,22 @@
+@startuml
```

Comment:
Remove this file, it is irrelevant to the template.

Reply:
Done